src/Raven.Client/Documents/Changes/DatabaseChanges.cs

```diff
@@ -56,11 +58,26 @@ public DatabaseChanges(RequestExecutor requestExecutor, string databaseName, Act
             _task = DoWork();
         }
 
+        public static event Func<object, X509Certificate, X509Chain, SslPolicyErrors, bool> WebSocketsRemoteCertificateValidationCallback;
+
+        private static bool OnWebSocketsRemoteCertificateValidationCallback(object sender, X509Certificate certificate, X509Chain chain, SslPolicyErrors errors)
+        {
+            if (WebSocketsRemoteCertificateValidationCallback == null)
+                return errors == SslPolicyErrors.None;
+
+            return WebSocketsRemoteCertificateValidationCallback(sender, certificate, chain, errors);
+        }
+
         public static ClientWebSocket CreateClientWebSocket(RequestExecutor requestExecutor)
         {
             var clientWebSocket = new ClientWebSocket();
             if (requestExecutor.Certificate != null)
                 clientWebSocket.Options.ClientCertificates.Add(requestExecutor.Certificate);
+
+#if NETCOREAPP2_1
```

Comment:
change to NETCOREAPP, we will not have to remember to change this when 2_2 or 3_0 will be out.